ui/src/app/shared/components/resource-submit.tsx

```diff
@@ -26,6 +27,14 @@ export class ResourceSubmit<T> extends React.Component<ResourceSubmitProps<T>, R
                 <Formik
                     initialValues={{resource: this.props.defaultResource, resourceString: jsYaml.dump(this.props.defaultResource)}}
                     onSubmit={(values, {setSubmitting}) => {
+                        if (this.props.validate !== undefined) {
```

Comment:
This is the same thing:
```suggestion
                        if (this.props.validate) {
```